spec/controllers/api/v1/interactive_pages_controller_spec.rb

```diff
@@ -734,5 +734,30 @@ def add_interactive_to_section(page, interactive, section)
     end
   end
 
+  describe "#get_portal_list" do
+    it "returns an array of OAuth portals including each portal's name and authorization path" do
+      Api::V1::InteractivePagesController.any_instance.stub(:user_omniauth_authorize_path).and_return('/auth/foo')
+      xhr :get, "get_portal_list"
+      expect(response.status).to eq(200)
+      expect(response.body).to eql({
+        success: true,
+        portals: [
+            {
+              name: "Portal",
+              path: "/auth/foo"
+            },
+            {
+              name: "Concord.Portal",
+              path: "/auth/foo"
+            },
+            {
+              name: "Portal",
+              path: "/auth/foo"
+            }
```

Comment:
This duplication of the data in lines 745-748 is partially the result of [line 54 in spec/spec_helper.rb](https://github.com/concord-consortium/lara/blob/79ec90befaaf1d4b2c6ddc190bfefdbeaebb398d/spec/spec_helper.rb#L54). I'm not sure, but it seems like that line should be changed to use something other than "portal" as the strategy name value. There's already a portal named Portal which seems to exist by default in the testing environment. 

I tried both removing line 54 in spec/spec_helper.rb and then changing the strategy name instead, but both options caused other problems without a simple solution. So I've left it as is even though it makes the response here look a little odd.